### ISSUE TITLE
[Serious] Focus indicator lacks 3 to 1 contrast ratio

### ISSUE THREAD

**Issue by ntallman** [APTrust/dart#577]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87600 |
| **Checkpoint** | 1.4 Distinguishable › Non-Text Contrast - States of User Interface Components (1.4.11.b) |
| **Impact** | Serious |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 01 Settings Workflow |
| **URL** | <https://aptrust.org/documentation-page/dart-digital-archivists-resource-tool/> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Date Reported** | 05/06/26 at 4:23 pm |
| **Issue Status** | Open |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | kim.casey@ablr360.com |

## Description

Focus indicator lacks 3 to 1 contrast ratio.
Pattern: Buttons have (depending on the color of the button) light focus indicators.  For example, the blue buttons, have a light blue focus indicator.  When contrasted with the white background, this is difficult to see.
See Settings User Experience video at 1:45.

## Recommendation

Rule : 
The contrast of visual focus indicators against the background MUST be at least 3 to 1.

How To Fix: 
Fix this issue by adjusting the visual focus indicator to achieve 3 to 1 contrast according the scenarios below.

When a focus indicator appears:

1. Outside the component, it needs to contrast with the background that the component is on.
2. Inside the component, it needs to contrast with the adjacent color(s) within the component.
3. As the border of the component (inside the component and adjacent to the outside), it needs to contrast with both adjacent colors.
4. Appears partly inside and partly outside, either part of the focus indicator can contrast with the adjacent colors.

Reference: 
WCAG Understanding document: https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast

Background: 
People who have low vision or are colorblind may have difficulty perceiving that an element is interactive or what its state is - e.g. whether it has keyboard focus, is selected/checked/pressed, etc. - if the contrast between the element boundaries and/or state indicators and its background or adjacent colors is insufficient. When an interactive element's visual boundaries and state indicators have adequate contrast, people who have low vision or are colorblind are more likely to be able to perceive which elements on the screen are interactive and what their current state is.

## Screenshots

![01_focus buttons](https://raw.githubusercontent.com/APTrust/dart/master/docs/accessibility-audit/screenshots/87600/01_focus_buttons.png)
